### ISSUE TITLE
Feature request: Configuration 'presets' for specific tasks

### ISSUE THREAD

**Issue by deduijk** [NECROBOTIO/NecroBot#3583]
It would be great if  you could select a configuration profile at bot start;
for example:

- farm only stops
- default
- only catch rare/superare
- max XP farming
- Max candy/ stardust farming
-...

Etc.

That way if you set out for a specific task you don't have to copy, rename change values , run bot, delete and rename old to current etc.

**Comment by schwartzadev** [comment]
You mean that you would create, say 4, different configs and on start, necrobot would ask which profile you want to use for the given session?

**Comment by deduijk** [comment]
Yes; although it would be nice if there were some example configs shipping with necro as well.

**Comment by wyvern19** [comment]
yes, we could work with a set of configs files.  Perhaps also mix in a location with the config so we can start base on location / task - I would assume a read from the config directory and then a list of all json config files using the name of the file in numbered order... 1,2,3,4 how ever many files and just type in the number to kick off that preset.

**Comment by deduijk** [comment]
depending on the update script used to update the config fiels when a new release comes we might need release # in the config file too.Then you have the possibility to exclude not updated configs.

OR

Make config release folders: only configs in the matching config release folder to release version are shown.
